components/apimgt/org.wso2.carbon.apimgt.tracing/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    ~ Copyright (c) 2009-2011, WSO2 Inc. (http://www.wso2.org) All Rights
+    Reserved. ~ ~ Licensed under the Apache License, Version 2.0 (the
+    "License"); ~ you may not use this file except in compliance with the
+    License. ~ You may obtain a copy of the License at ~ ~
+    http://www.apache.org/licenses/LICENSE-2.0 ~ ~ Unless required by
+    applicable law or agreed to in writing, software ~ distributed under
+    the License is distributed on an "AS IS" BASIS, ~ WITHOUT WARRANTIES
+    OR CONDITIONS OF ANY KIND, either express or implied. ~ See the
+    License for the specific language governing permissions and ~
+    limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<parent>
+		<groupId>org.wso2.carbon.apimgt</groupId>
+		<artifactId>apimgt</artifactId>
+		<version>6.4.42-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>org.wso2.carbon.apimgt.tracing</artifactId>
+	<packaging>bundle</packaging>
+	<name>WSO2 Carbon - API Tracing</name>
+	<description></description>
+	<url>http://wso2.org</url>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.wso2.carbon.identity.framework</groupId>
+			<artifactId>org.wso2.carbon.identity.core</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>jcl-over-slf4j</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>log4j-over-slf4j</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.wso2.carbon.apimgt</groupId>
+			<artifactId>org.wso2.carbon.apimgt.impl</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>io.jaegertracing</groupId>
+			<artifactId>jaeger-core</artifactId>
+			<version>0.30.0</version>
```

Comment:
Versions of the dependency should be declared in the root pom, and inherited here. Check how other dependencies were added, and follow the same steps.

Reply:
fixed